src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -65,10 +65,11 @@ public static bool IsZipDeployment(this IEnvironment environment)
             // Run From Package app setting exists
             return IsValidZipSetting(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteZipDeployment)) ||
                 IsValidZipSetting(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteAltZipDeployment)) ||
-                IsValidZipSetting(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage));
+                IsValidZipSetting(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage)) ||
+                IsValidZipSetting(environment.GetEnvironmentVariable(EnvironmentSettingNames.ScmRunFromPackage));
         }
 
-        private static bool IsValidZipSetting(string appSetting)
+        public static bool IsValidZipSetting(string appSetting)
         {
             // valid values are 1 or an absolute URI
             return string.Equals(appSetting, "1") || Uri.TryCreate(appSetting, UriKind.Absolute, out Uri result);
```

Comment:
SCM_RUN_.. doesnt support 1 (yet).

Reply:
Will separate out logic for scm so that it only checks for valid url.